pkg/render/fluentd.go

```diff
@@ -137,6 +138,29 @@ type fluentdComponent struct {
 	installation    *operatorv1.InstallationSpec
 	clusterDomain   string
 	osType          OSType
+	linuxImage      string
+	windowsImage    string
```

Comment:
could this just be `image` and then set it to the correct one based on osType inside of ResolveImages?